routes/membership.js

```diff
@@ -0,0 +1,123 @@
+
+stripe = require("stripe")("sk_test_BQokikJOvBiI2HlWgH4olfQ2");
+module.exports = function(app) {
+
+    app.get('/membership/create', function(req, res, next) {
+        console.log("test");
+        res.render('payment/paywall.jsx');
+    });
+
+
+
+//single token that represents the info the customer has
+{
+  id: "tok_u5dg20Gra", // String of token identifier,
+  card: {...}, // Dictionary of the card used to create the token
+  created: 1447356796, // Integer of date token was created
+  currency: "usd", // String currency that the token was created in
+  livemode: true, // Boolean of whether this token was created with a live or test API key
+  object: "token", // String identifier of the type of object, always "token"
+  used: false // Boolean of whether this token has been used
+}
+
+//sending form to the server
+function stripeResponseHandler(status, response) {
```

Comment:
i'm assuming this is copy/pasted code as well. this is code that you'd use on the front-end if you were doing a custom implementation of the stripe button. since we're using stripe checkout, i don't think we need this.

`$` is a jQuery reference, which is a front-end JavaScript framework.